src/core/text/Text.js

```diff
@@ -319,8 +319,8 @@ export default class Text extends Sprite
         {
             const trimmed = trimCanvas(canvas);
 
-            canvas.width = trimmed.width;
-            canvas.height = trimmed.height;
+            canvas.width = trimmed.width(trimmed.width);
+            canvas.height = trimmed.width(trimmed.height);
```

Comment:
Can you explain and give context for this change? It doesn't make sense to me. Should it be `expandToEvent(trimmed.height)`?

Reply:
You are right, it is my mistake. Sorry. expandToEven is correct.

I think about texture extending, it is will add one pixel at right side, and all right aligned text may look not axactly aligned.

Maybe better will be to fix anchor applying? Round anchor's shifting:
Sprite.js
```
            w1 = trim.x - Math.round(anchor._x * orig.width);
            w0 = w1 + trim.width;

            h1 = trim.y - Math.round(anchor._y * orig.height);
            h0 = h1 + trim.height;
```
I tried this aproach, fixes blureness perfectly too.

Reply:
Isn't that effectively enabling 'roundToPixels' on the display object?

Reply:
No, its not. RoundPixels is implemented wrong way in pixi and its very hard to fix it. 

@Megabyteceer idea looks better because its very stupid and it works.

Reply:
Math.round for anchor only is better, because you can still move object smoothly by x+=0.1 on subpixes speed, and object is sharp when it is stops at rounded x y.